recipes/agate-lookup/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{%set name = "agate-lookup" %}
+{%set version = "0.3.1" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "0b0de078b1e56a88a37983f59d0e911272199ed08cebe88ed99390ae8cc12073" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - agate >=1.5.0
+    - requests >=2.9.1
+    - pyyaml >=3.11
```

Comment:
Can you try removing the `run` time dependencies from the `build` section and see how it goes?